src/python_testing/matter_testing_support.py

```diff
@@ -273,6 +274,34 @@ def name(self) -> str:
         return self._name
 
 
+class EventChangeCallback:
+    def __init__(self, expected_cluster: ClusterObjects):
+        self._q = queue.Queue()
+        self._expected_cluster = expected_cluster
+
+    async def start(self, dev_ctrl, node_id: int, endpoint: int):
+        self._subscription = await dev_ctrl.ReadEvent(node_id,
+                                                      events=[(endpoint, self._expected_cluster, True)], reportInterval=(1, 5),
+                                                      fabricFiltered=False, keepSubscriptions=True, autoResubscribe=False)
+        self._subscription.SetEventUpdateCallback(self.__call__)
+
+    def __call__(self, res: EventReadResult, transaction: SubscriptionTransaction):
+        if res.Status == Status.Success and res.Header.ClusterId == self._expected_cluster.id:
+            logging.info(
+                f'Got subscription report for event on cluster {self._expected_cluster}: {res.Data}')
+            self._q.put(res)
+
+    def WaitForEventReport(self, expected_event: ClusterObjects.ClusterEvent, timeout: int = 10):
```

Comment:
Please use pep8 snake_case style for methods, arguments and local variables

Reply:
Changed